src/java/main/org/apache/zookeeper/server/NIOServerCnxn.java

```diff
@@ -819,7 +819,7 @@ public void commandRun() {
             }
         }
 
-        private void print(String key, long number) {
+        private void print(String key, Object number) {
```

Comment:
It's better to override print with double number, which will be more efficient, as we don't need to cast all the numbers to Object while printing the metrics.